terraform/automation-account.tf

```diff
@@ -0,0 +1,9 @@
+
+resource "azurerm_automation_account" "vh_infra_core_ado" {
+  name                = "vh-infra-ado-auto"
```

Comment:
Can you change to vh-infra-core-ado to keep in line with other resources

(I know this naming convention is bad but is just to keep in line with others already there)